package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "opentelemetry-base",
+  "version": "0.0.1",
+  "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "scripts": {
```

Comment:
There should be a script to do TDD in watch mode. I find this is usually very hard to add later on, especially in TypeScript projects that take an increasing amount of time to build.

Reply:
Sure, done. `"tdd": "yarn run test -- --watch-extensions ts --watch"`